mqt/qcec/parameterized.py

```diff
@@ -0,0 +1,174 @@
+"""Functionality for checking equivalence of parameterized quantum circuits."""
+
+from __future__ import annotations
+
+from itertools import chain
+from typing import Any
+
+import numpy as np
+from mqt.qcec import Configuration, EquivalenceCheckingManager
+from qiskit import QuantumCircuit
+from qiskit.circuit import Parameter, ParameterExpression
+
+
+def __ecm_from_config_or_kwargs(
+    circ1: QuantumCircuit, circ2: QuantumCircuit, configuration: Configuration | None = None, **kwargs: Any
+) -> EquivalenceCheckingManager:
+    if kwargs:
+        # create the equivalence checker from keyword arguments
+        return EquivalenceCheckingManager(circ1, circ2, **kwargs)
+    else:
+        if configuration is None:
+            configuration = Configuration()
+
+        # create the equivalence checker from configuration
+        return EquivalenceCheckingManager(circ1, circ2, configuration)
```

Comment:
At some point (not this PR), this should probably be unified in some place so that the other `verify` functions can make use of it too. 
Technically, that function could be exposed as part of the bindings. (a class method of the ECM `from_config_or_kwargs`). Now that I think of it: wouldn't that also be a better way for the, currently overloaded, regular constructors? Overloading in Python isn't that great, right?

Reply:
I think the current method is actually adequate. We can simply have a `utils.py` that all files can use

Reply:
Actually, this will be resolved by fixing Issue  #175